src/GitLabApiClient/Models/RateLimitPagedInfo.cs

```diff
@@ -0,0 +1,9 @@
+
+namespace GitLabApiClient.Models;
+
+public class RateLimitPagedInfo
```

Comment:
nit: for consistency, I would rename to `RateLimitPagingInfo` (or rename `PagingInfo` to `PagedInfo` but I personally prefer `Paging`)

Reply:
Updated